### PR TITLE
registering backstage time saver plugin

### PR DESCRIPTION
**Pull request by tduniec**
## Hey, I just made a Pull Request!

Introducing timeSaver plugin. It does some analysis graphs and charts that helping your management and team how using scaffolder templates can bring value and savings to the onboarded Teams!!!

It is declarative so it requires to put some savings data for each template execution into the template definition.  [Checkout README](./time-saver-backend/README.md)

byTemplate
tsAllStats
tsAllStats2
tsByTeam
tsByTeam2

#### :heavy_check_mark: Checklist

- ➖  A changeset describing the change and affected packages. ([more info](git)) // **please let me know if this is needed on plugin init**
- ✔️  Added or updated documentation
- ➖  Tests for new functionality and regression tests for bug fixes
- ✔️  Screenshots attached (for UI changes)
- ✔️ All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

Please let me know if any changes are needed and what else should be accomplished moving forward publishing this plugin.

### CODE REVIEW

**Review by tudi2d — CHANGES_REQUESTED** [review]
Thanks for adding your new plugin & welcome to the community! Make sure you follow the tips for submitting your plugin: https://backstage.io/docs/plugins/add-to-directory#submission-tips. You would need to publish the seperate packages on NPM on your own.

as the link in the description of the PR does not work here the link for others looking at the PR: https://github.com/tduniec/backstage-timesaver-plugin

**Comment by tduniec** [comment]
Hey @tudi2d!

I have added few adjustments

File: microsite/data/plugins/time-saver.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: TimeSaver
+author: tduniec
+authorUrl: https://github.com/tduniec
+category: Discovery
+description: Visualise your time saved using Backstage Scaffolder templates
+documentation: https://github.com/tduniec/backstage-timesaver-plugin/blob/main/README.md
+iconUrl: img/tsLogo.png
```
Comment (tudi2d) [inline microsite/data/plugins/time-saver.yaml:8]: This image is still not part of the PR. Make sure to either add it in the PR or change this to an URL

**Review by tudi2d — CHANGES_REQUESTED** [review]
thanks - last thing to fix is the `iconUrl`

File: microsite/data/plugins/time-saver.yaml (hunk 1 above)
Comment (toma2395) [reply microsite/data/plugins/time-saver.yaml]: updated

**Review by tudi2d — APPROVED** [review]
